### PR TITLE
vendor g/g@729a1cce46be

### PR DESCRIPTION
**Pull request by vpnachev**
**How to categorize this PR?**

/area dev-productivity
/kind task
/priority normal
/platform aws

**What this PR does / why we need it**:
vendor g/g@729a1cce46be

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
I might change this PR to v1.8.0, in case it is released next week.
Also, I will follow up on the other providers with the same vendoring.

/invite @stoyanr 

**Release note**:

```improvement operator
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -97,7 +97,7 @@ revendor:
 	@GO111MODULE=on go mod tidy
 	@chmod +x $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/*
 	@chmod +x $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/.ci/*
-	@chmod +x $(REPO_ROOT)/vendor/github.com/gardener/gardener/extensions/hack/*
+	@chmod +x $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/*
```
Comment (rfranzke) [inline Makefile:100]: duplicate of line 98

File: hack/hook-me.sh [hunk 2]
```diff
@@ -14,4 +14,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-./vendor/github.com/gardener/gardener/extensions/hack/hook-me.sh aws $@
+./vendor/github.com/gardener/gardener/extensions/hook-me.sh aws $@
```
Comment (rfranzke) [inline hack/hook-me.sh:17]:
```suggestion
./vendor/github.com/gardener/gardener/hack/hook-me.sh aws $@
```

Comment (vpnachev) [reply hack/hook-me.sh]: done

**Pull request merged**